src/NuGet.Core/NuGet.Protocol/Resources/PackageMetadataResourceV3.cs

```diff
@@ -30,6 +37,16 @@ public class PackageMetadataResourceV3 : PackageMetadataResource
             _packageDetailsUriResource = packageDetailsUriResource;
         }
 
+        /// <summary>
+        /// Query nuget package list from nuget server for Consolidate UI. This implementation optimized for performance so instead of keeping gian JObject in memory we use strong types.
```

Comment:
```suggestion
        /// Query nuget package list from nuget server for Consolidate UI. This implementation optimized for performance, so instead of keeping giant JObject in memory, we use strong types.
```

Reply:
Fixed.